app/models/user.rb

```diff
@@ -39,6 +39,21 @@ def count_attended
     events.size
   end
 
+  def self.count_points(user_id)
+    current_event_types = EventTypes.all
+    point_hash = {}
+    current_event_types.each do |event_type|
+      point_hash[event_type.id] = event_type.pointValue
+    end
+    this_users_events = Event.joins(:events_users)
```

Comment:
This is already implemented as `user.events` and `user.events_users`

Reply:
Does it though? I didn't see those using event types

Reply:
`this_users_events` is; I'm making a branch `point-filtering` to add filtering to this function real quick so I can refactor it as well